src/atlas4py/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.35.1.dev15"
+__version__ = "0.39.0.dev16"
```

Comment:
```suggestion
__version__ = "0.39.0.dev0"
```
Why not restart from 0